### PR TITLE
US024.2 - Authorisation

### PR DESCRIPTION
**Pull request by michaltangri**
Acceptance criteria:

Admin must have access to employee view and admin view.
Non-admin user will only have access to employee view.
Non-logged in users will not be able to access any pages.

### CODE REVIEW

**Review by sebastianczajkowski — APPROVED** [review]
I didn't find any errors

File: src/main/java/com/kainos/ea/backend/interceptors/AuthInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            throws Exception {
+        // TODO Auto-generated method stub
+
+    }
+
+    @Override
+    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler, ModelAndView modelAndView)
+            throws Exception {
+        // TODO Auto-generated method stub
+
```
Comment (mactrz) [inline src/main/java/com/kainos/ea/backend/interceptors/AuthInterceptor.java:33]: Should these comments stay here?

**Review by mactrz — APPROVED** [review]
Everything else looks great, nice work!

File: src/main/java/com/kainos/ea/backend/interceptors/AuthInterceptor.java (hunk 1 above)
Comment (michaltangri) [reply src/main/java/com/kainos/ea/backend/interceptors/AuthInterceptor.java:31]: I've removed them, thanks!

**Pull request merged**